schema/deploy/tables/project_revision.sql

```diff
@@ -6,7 +6,8 @@ create table cif.project_revision (
   id integer primary key generated always as identity,
   project_id integer references cif.project(id),
   change_status varchar(1000) default 'pending' references cif.change_status,
-  change_reason varchar(10000)
+  change_reason varchar(10000),
+  is_first_revision boolean default false
```

Comment:
Would this be something we could calculate instead? Do we need to store that flag in the database?